packages/web-components/src/kumo/anchor-button/anchor-button.ts

```diff
@@ -0,0 +1,352 @@
+import { attr, FASTElement, Observable } from '@microsoft/fast-element';
+import { keyEnter } from '@microsoft/fast-web-utilities';
+import { StartEnd } from '../../patterns/index.js';
+import type { StartEndOptions } from '../../patterns/index.js';
+import { applyMixins } from '../../utils/apply-mixins.js';
+import { toggleState } from '../../utils/element-internals.js';
+import {
+  AnchorAttributes,
+  type AnchorButtonAppearance,
+  type AnchorButtonShape,
+  type AnchorButtonSize,
+  type AnchorTarget,
+} from './anchor-button.options.js';
+
+/**
+ * Anchor configuration options
+ * @public
+ */
+export type AnchorOptions = StartEndOptions<AnchorButton>;
+
+/**
+ * An Anchor Custom HTML Element.
+ * Based largely on the {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element }.
+ *
+ * @slot start - Content which can be provided before the anchor content
+ * @slot end - Content which can be provided after the anchor content
+ * @slot - The default slot for anchor content
+ * @csspart control - The anchor element
+ * @csspart content - The element wrapping anchor content
+ *
+ * @public
+ */
+export class BaseAnchor extends FASTElement {
```

Comment:
Can't we just import base anchor from fluent?

Reply:
Removed this file entirely and am just importing `AnchorButton`... but looking at it.. I probably need to make my own class because we don't need `shape`

Reply:
Wonder if shape should just move outside base anchor in fluent?

Reply:
Shape is in `AnchorButton` but not in `BaseAnchor`

Reply:
Yeah, I need create my own classes for all of these that extend the base class. It will be a dupe of the Fluent element's class but without `shape`. Will get that in and it should be ready to go.